scripts/types-mom/apis.d.ts

```diff
@@ -157,6 +157,12 @@ declare namespace MomentumPlayerAPI {
 
 	/** Gets whether the player is ducking. */
 	function IsDucking(): boolean;
+
+	/** Gets whether the player is sprinting. */
+	function IsSprinting(): boolean;
+
+	/** Gets whether the player is walking. */
+	function IsWalking(): boolean;
```

Comment:
These should maybe mention that this is ahop only, especially walking since other modes can walk but its not quite the same thing.